pkg/flavor/kubernetes/flavor.go

```diff
@@ -590,3 +590,20 @@ func (flavor *Flavor) GetCredentialsFromPodSpec(volumeHandle string, podName str
 	}
 	return nil, fmt.Errorf("Pod %s/%s does not contain the volume %s", namespace, podName, volumeHandle)
 }
+
+// GetFsTypeOfPvName retrieves volume filesystem for a given CSI volname
+func (flavor *Flavor) GetFsTypeOfPvName(name string) (string, error) {
```

Comment:
Let's not limit ourselves to one property.  What if we make this more generic so it can apply to any property we might want to read?  Something like:

`GetPropertyOfPv(property, pvName string)`

Reply:
Sorry, I didnt understand by all property of PV,  shouldn't we limit ourselves to PV's CSI property or VolumeAttributes only even if we want to make it more generic?

Reply:
Made it generic for volumeAttribute properties.